### DISCUSSION TITLE
Installation problem under HA

### DISCUSSION THREAD

**Discussion by psylapse** [flobz/psa_car_controller#606] [General]
I hate to ask such a foolish question, but installation systematically fails with

Can't install flobz/psa_car_controller:v3.0.12: 404 Client Error for http+docker://localhost/v1.42/images/flobz/psa_car_controller:v3.0.12/json: Not Found ("No such image: flobz/psa_car_controller:v3.0.12")

System details

 Home Assistant 2023.7.2 
Supervisor 2023.08.1 
Operating System 10.3 
Frontend 20230705.1 - latest 
Lovelace etc

My newbie (to HA) guess is that I have to have HA installed under docker? in a SSH shell, docker inspect simply gives docker not found

**Comment by gevans3342** [comment]
Hi There,,

Did you ever find a solution to this? As I'm having exactly the same issue.....

**Reply by psylapse** [reply]
Sadly no solution and have given up trying. Just taken delivery of aPi5 and will try a completely fresh build. If that works, I will let you know

**Comment by gevans3342** [comment]
Ok, thanks for the update.

On Fri, 8 Dec 2023 at 16:20, psylapse ***@***.***> wrote:

> Sadly no solution and have given up trying. Just taken delivery of aPi5
> and will try a completely fresh build. If that works, I will let you know
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/flobz/psa_car_controller/discussions/606#discussioncomment-7801480>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2RRFNGJFCOTO4N63Q7FBMDYIM445AVCNFSM6AAAAAA4GYOWGWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMBRGQ4DA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
*Sent by Gmail*
